### ISSUE TITLE
Php-fpm listen address 

### ISSUE THREAD

**Issue by dpark77** [virtualmin/virtualmin-gpl#373]
When php-fpm is enabled in Server configuration -> php options

In nginx.conf,
it creates, 
fastcgi_pass localhost:8000;

Default listening address should be corrected to 127.0.0.1 instead of localhost.

Setting to localhost always has some nagging issues in production usage whereas 127.0.0.1 works all the time flawlessly.

-> fastcgi_pass 127.0.0.1:8000;

**Comment by jcameron** [comment]
Do you have an entry for `localhost` in your `/etc/hosts` file that doesn't map to 127.0.0.1 ?

**Comment by dpark77** [comment]
Yes mapped correctly in /etc/hosts but this is about php-fpm tcp/ip socket settings.

For a tcp/ip socket, listening address should be an ip address 127.0.0.1:(port #) as a default.

Something like Localhost:8000 works fine out of the box initially, but it'll run into some nagging issues later because localhost is not an ip address.
Btw, /etc/hosts is strictly for hostname mapping, not for tcp/ip:port mapping.

Virtualmin creates some [random numbers.conf] for subserver's php-fpm tcp/ip socket
e.g. 123456789.conf (www.conf)

There should be 127.0.0.1:8000 in there as well instead of localhost:8000.

I already tested that 
locahost and 127.0.0.1 can't be shared nor used together in tcp/ip socket setup, it simply would not work.

**Comment by jcameron** [comment]
Ok, I will update Virtualmin to use 127.0.0.1 instead of localhost for these connections..

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
